pkg/cli/connect.go

```diff
@@ -14,21 +14,28 @@ var validFileToFileEdges = []string{
 	"dependencies",
 }
 
+var validFileToPackageEdges = []string{
+	"targets",
+}
+
 var connectCmdFlags = struct {
-	fileToFileEdge string
+	fileToFileEdge    string
+	fileToPackageEdge string
 }{}
 
 func init() {
 	rootCmd.AddCommand(connectCmd)
 	connectCmd.Flags().StringVar(&connectCmdFlags.fileToFileEdge, "fileToFileEdge",
 		"derivedFrom", fmt.Sprintf("Edge to use when connecting FileNode to FileNode. One of %v", validFileToFileEdges))
+	connectCmd.Flags().StringVar(&connectCmdFlags.fileToPackageEdge, "fileToPackageEdge",
```

Comment:
Why don't you just use one flag for the edges "--edge" ?

Reply:
That is a nice idea